app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvUploadForm
+  include ActiveModel::Validations
+
+  attr_reader :keyword_ids
+
+  validates :file, presence: true, valid_keyword_file: true
+
+  def save(file)
+    @file = file
+    return false unless valid?
+
+    begin
+      # rubocop:disable Rails/SkipsModelValidations
+      @keyword_ids = Keyword.insert_all(parsed_keywords).pluck('id')
```

Comment:
Just curious how will you plan to start the search for each keyword in the next PR, will you have a cronjob to check the new added record or how, because with the `insert_all` there will be no callback event like `after_commit` in the model https://guides.rubyonrails.org/active_record_callbacks.html#skipping-callbacks

Reply:
yeah I intend to ask you about this. 

I'm thinking about using create
```
# from
@keyword_ids = Keyword.insert_all(parsed_keywords).pluck('id')

# to
@keyword_ids = Keyword.create(parsed_keywords).pluck('id')
```

I see that it performs a SQL query per item. Should I worry about it? Not that I'll affect this project much, just wonder about the general way to approach this.

Reply:
Regarding the performance, we shouldn't worry it for now, just let's just insert 1 by 1 keyword for now, so with that you can use the ActiveRecord callbacks (after_commit, after_create) to trigger the background job later

Reply:
great, got it. Fixed in 9162b42eee632a76a818c3ded71d0bb68ccee90a